runtime/server/generate_metrics_view.go

```diff
@@ -403,55 +395,32 @@ type metricsViewYAML struct {
 }
 
 type metricsViewDimensionYAML struct {
-	Label       string
-	Column      string
-	Description string
+	Label  string
+	Column string
 }
 
 type metricsViewMeasureYAML struct {
-	Name                string
-	Label               string
-	Expression          string
-	Description         string
-	FormatPreset        string `yaml:"format_preset"`
-	ValidPercentOfTotal any    `yaml:"valid_percent_of_total"`
+	Label       string
+	Expression  string
+	Name        string
+	Description string
```

Comment:
Subjective, but I really like having `name` first – kind of like a variable declaration (name = props) or `id` attribute in HTML

Reply:
Ye agree, I'll move both column and name back to first position